### ISSUE TITLE
Add possibility to add a reviewer for a GitHub Pull Request in VS GitHub Extension

### ISSUE THREAD

**Issue by chund** [github/VisualStudio#2281]
I would appreciate if you could add the possibility to add a reviewer for a GitHub Pull Request directly from within Visual Studio.

Currently you need to create the PR in VS. Then open the PR in GitHub to add the reviewer.

Adding a reviewer was easily possible in TFS. Would be cool if the GitHub Extension could offer the same feature.

**Comment by chund** [comment]
I already addressed this at VisualStudio but were told to post here:
https://developercommunity.visualstudio.com/content/idea/489543/add-possibility-to-add-a-reviewer-for-a-github-pul.html

**Comment by chund** [comment]
I am also referring to this, since pull requests are the Microsoft recommended way to do Code Reviews with Git:

https://docs.microsoft.com/en-us/azure/devops/repos/git/pull-requests

**Comment by meaghanlewis** [comment]
👋 @chund thanks for opening up this issue here! This is the best place for us to track enhancements like this. 

We are a small team and will try to get around to adding this to the extension. Additionally, you are more than welcome to work on this enhancement yourself and submit a pull request.

**Comment by gzuckerman** [comment]
@meaghanlewis Would this be open for anyone to pick up?

**Comment by meaghanlewis** [comment]
@gzuckerman yes, it is! We would welcome any contributions and our developers will review any pull requests that are submitted
